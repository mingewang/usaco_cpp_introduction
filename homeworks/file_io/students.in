Alice 84 90 88
Bob 69 75 80
Charlie 94 85 92