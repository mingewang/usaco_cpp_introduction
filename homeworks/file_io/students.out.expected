Alice: 86.67
Bob: 74.00
Charlie: 89.67